react/modules/extraEvents.ts

```diff
@@ -1,9 +1,7 @@
 import push from './push'
-import {
-  PixelMessage,
-} from '../typings/events'
+import { PixelMessage } from '../typings/events'
 
-export function sendExtraEvents(e: PixelMessage) {
+export async function sendExtraEvents(e: PixelMessage) {
```

Comment:
Why add this `async` to the function?

Reply:
I made all the event functions async so one won't prevent the others from running right away. I don't know if this can lead to any perceivable difference, what do you think?

Reply:
But you're not returning a Promise or `await`ing on anything. In this case, what's the effect of making the function async?

Reply:
When I make it async doesn't it automatically returns a promise of the result? Even If it's a void/null?
I thought that by making it async the three main event functions would execute in parallel.

Reply:
Yep! You're right! My bad 😅